src/types/styles.ts

```diff
@@ -25,6 +28,25 @@ export enum InfillStyle {
   // DYNAMIC = 4,
 }
 
+const Slic3rInfillStyles = {
+  Rectilinear: 0,
+  Triangles: 1,
+  Gyroid: 2,
+  AdaptiveCubic: 3,
+  Honeycomb3D: 4,
+  Lightning: 5,
+};
+type Slic3rInfillStyle = ValueOf<typeof Slic3rInfillStyles>;
+
+export const slic3rInfillStylesToFillPattern = {
+  [Slic3rInfillStyles.Rectilinear]: InfillPattern.RECTILINEAR,
+  [Slic3rInfillStyles.Triangles]: InfillPattern.TRIANGLES,
+  [Slic3rInfillStyles.Gyroid]: InfillPattern.GYROID,
+  [Slic3rInfillStyles.AdaptiveCubic]: InfillPattern.ADAPTIVE_CUBIC,
+  [Slic3rInfillStyles.Honeycomb3D]: InfillPattern.HONEYCOMB_3D,
+  [Slic3rInfillStyles.Lightning]: InfillPattern.LIGHTNING,
+} as const;
```

Comment:
Can we add `satisfies Record<Slic3rInfillStyles, InfillPattern>`?

Reply:
I defined the var to be `Record<Slic3rInfillStyles, InfillPattern>`

Reply:
If you want to keep the `as const` you can type it as `readonly Record<Slic3rInfillStyles, InfillPattern>`